app/controllers/dashboard_controller.rb

```diff
@@ -85,6 +120,39 @@ def select
     end
   end
 
+  def getHangoutsLink
```

Comment:
Style: could you please change this to `get_hangouts_link` to stay consistent with the convention?  You may also need to update some other places.

Reply:
Done.